README.md

```diff
@@ -1 +1,173 @@
-# capstone-project-1
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>ArtWork Gallery</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
```

Comment:
- Hey, since you've not added this section in the readme body, please, kindly remove it from the list of navigation to make your readme contain only content relevant to the project making your readme more professional.